UnityProject/Assets/Scripts/Messages/Client/Interaction/RequestInteractMessage.cs

```diff
@@ -51,6 +51,8 @@ public class RequestInteractMessage : ClientMessage
 	public int SlotIndex;
 	//named slot targeted in storage
 	public NamedSlot NamedSlot;
+	// connections used in CableApply
```

Comment:
As in the MouseInputController I think this is meant to be a generic interaction class without item specific logic in it. If structured correctly you should be either able to use one of the existing methods, or create a new generic one that can be reused if none are appropriate rather than having a cable specific one.

Reply:
I have transferred this functionality to a new script (CablePlacementVisualisation)